src/main/java/org/graylog2/plugins/slack/SlackPluginMetadata.java

```diff
@@ -10,6 +10,8 @@
 import java.util.Set;
 
 public class SlackPluginMetadata implements PluginMetaData {
+    private static final String PLUGIN_PROPERTIES = "org.graylog.plugins.graylog-plugin-slack/graylog-plugin.properties";
```

Comment:
Since the current Maven build doesn't use [resource filtering](https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html), the property file won't contain the information you probably think it does.